java/vector/src/main/java/org/apache/arrow/vector/FieldVector.java

```diff
@@ -90,4 +90,11 @@ public interface FieldVector extends ValueVector {
    * @return buffer address
    */
   long getOffsetBufferAddress();
+
+  /**
+   * Sets the value at index to null.
```

Comment:
Seems like other docstrings use the infinitive:
```suggestion
   * Set the element at the given index to null.
```